frontend/src/components/Graph/index.tsx

```diff
@@ -135,6 +136,10 @@ function Graph({
 					},
 					tooltip: {
 						callbacks: {
+							title(context) {
+								const date = new Date(context[0].parsed.x);
+								return format(date, 'MMM dd yyyy, HH:mm:ss');
```

Comment:
```suggestion
								return format(date, 'MMM dd yyyy, HH:mm:ss aaa');
```

Reply:
Hi @palashgdev! Thank you for the review.
Happy to make the change but wouldn't it be redundant to use am or pm in 24h time format?

Reply:
then can we make 12 h format ?

Reply:
Do you mean to convert the x-axis time format to 12h format instead? Sure, we can.

Reply:
yup it would be great